prow/config.yaml

```diff
@@ -1007,10 +1003,6 @@ periodics:
       env:
```

Comment:
Please add `--gce-ssh` and `--gce-pub` flags to the args section.

Also please add a `-name: USER value: ubuntu` or something to the `env` section.

I believe that will fix up these jobs!